test/functional/plugin/lsp_spec.lua

```diff
@@ -861,4 +861,27 @@ describe('LSP', function()
       ]])
     end)
   end)
+  describe('lsp.util.show_line_diagnostics', function()
+    it('show_line_diagnotics shows diagnostics for current line', function()
+      eq(1, exec_lua [[
+        BUFFER = vim.api.nvim_create_buf(false, true)
+        vim.api.nvim_buf_set_lines(BUFFER, 0, -1, false, {
+          "testing";
+          "123";
+        })
+        local diagnostics = {
+          {
+            range = {
+              start = { line = 0; character = 1; };
+              ["end"] = { line = 0; character = 3; };
+            };
+            message = "Syntax error";
```

Comment:
adding this here lets the test get farther:

    severity = require'vim.lsp.protocol'.DiagnosticSeverity.Error;